.wiki/_DV/Laws/StandardOperatingProcedure.txt

```diff
@@ -420,3 +420,30 @@ In order for a medical evaluation to be considered lawfully applicable, the CMO
 
 ==Quarantine==
 Medical staff are afforded the right to quarantine individuals that currently suffer from infectious or contagious diseases, and may confine them within pre-designated quarantine areas in the course of treatment, until the point that they are no longer infectious or harmful to the rest of the crew.
+=Engineering Regulations=
+===Wiring===
+All wiring must be concealed from public for their safety.
+In non public areas, catwalk is recommended to be placed for safety and mobility of workers.
+
+Wiring is to be held straight and tidy, without unnecessary looping, to avoid clumping, short circuiting and trpping hazard for the crew.
+==Atmospheric Regulations==
+===Habitable Rooms===
+Rooms with purpose of supporting organic life require the following:
+* Air pressure around 1 atmosphere (101.325 kPa)
+* Air temperature between 15° and 35° Celsius.
+* Barrier made of 2 meters (3.28 ft) of reinforced glass or stronger from non habitable space.
+* Air mixture supporting majority of station's crew.
+* Presence of hazardous gasses below 1 mol.
+
+===Pressure In Pipe Networks===
+It is recommended to set taget pressure in any pipe network double than one atmosphere (202.65 kPa).
+
+Disconnetcing pipes with pressure way above 2 atmospheres or containing hazardous gases above 1 mol, requires total disposal of hazardous gases and lowering pressure to 1 atmosphere or less.
+===Burn Chamber===
+Burn chamber specified for holding flamable gases and conducting their burn are required to have the following:
+* Release mechanism to let any object or gasses out into the environment, without endangering the station or its crew.
+* Barrier equal to or stronger than 2 meters (3.28 ft) of reinforced glass.
+
+All burn chambers must not have the following:
+* Airlocks directly leading to station's habitable rooms.
+* Any windows as barriers to station's habitable rooms.
```

Comment:
vague, could be interpreted that you cant use windows in a burn chamber at all, reword to be about a space gap